src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -40,8 +40,14 @@ function getDrupalValue(arr) {
  * need it in the future to convert weird uris like
  * `entity:node/27` to something resembling a
  * relative url
+ *
+ * If the uri begins with "internal:" it will be stripped out.
  */
 function uriToUrl(uri) {
+  const internal = 'internal:';
+  if (uri.startsWith(internal)) {
+    return uri.substring(internal.length);
+  }
```

Comment:
Should we do this yet, or wait until we've got the "real" URIs from Elijah?

Reply:
I don't think this harms anything, and once we get those real URLs we may not have to do this at all.